cardano-node/src/Cardano/CLI/Tx.hs

```diff
@@ -277,8 +280,8 @@ nodeSubmitTx
       \p@Consensus.ProtocolRealPBFT{} -> liftIO $ do
```

Comment:
So for instance here this could be:
```
 \p@Consensus.ProtocolRealPBFT{} -> liftIO $ withIOManager $ \iocp -> do
          -- TODO: Update submitGenTx to use `ExceptT`
          traceWith stdoutTracer ("TxId: " ++ condense (Consensus.txId gentx))
          submitTx iocp targetSocketFp
                   (pInfoConfig (Consensus.protocolInfo p))
                   gentx
                   nullTracer -- stdoutTracer
```